ui/src/user/UserCreate.js

```diff
@@ -51,9 +51,13 @@ const UserCreate = (props) => {
   return (
     <Create title={<Title subTitle={title} />} {...props}>
       <SimpleForm save={save} variant={'outlined'}>
-        <TextInput source="userName" validate={[required()]} />
+        <TextInput
+          spellCheck={false}
+          source="userName"
+          validate={[required()]}
+        />
         <TextInput source="name" validate={[required()]} />
-        <TextInput source="email" validate={[email()]} />
+        <TextInput spellCheck={false} source="email" validate={[email()]} />
         <PasswordInput source="password" validate={[required()]} />
```

Comment:
Don't we also need this in Password inputs?

Reply:
Spellcheck isn't enabled on `type="password"` inputs by default. This issue only affects password-inputs which use a "Show password" toggle to change the input to `type="text"`. I don't believe the Navidrome inputs do this, so there isn't a problem here AFAIK :nerd_face:

Reply:
Nice catch, I missed the little "eye" icon, toggling does indeed convert the password-input to a text-input.
I've amended create & edit user password fields (e0fabdab857d15c081d217f5857b9dc19cb7d471) :pray: